### PR TITLE
Notifications > Comment Details: show previous and next arrows in nav bar

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: #17790 

On the new Notification Comment details view, previous and next buttons are now displayed in the nav bar. 

They're not hooked up yet, so:
- For now tapping them simply logs a message (just to show they work).
- They are not disabled if there is no previous/next notification.

To test:

- Enable `notificationCommentDetails`.
- Go to Notifications > Comments > notification details.
  - The Edit button is always displayed (all devices, all orientations).
  - On an iPad, the arrow buttons are never displayed.
  -  On a regular size iPhone, the arrow buttons are always displayed.
  - On an X size iPhone:
    - In portrait, the arrow buttons are displayed.
    - In landscape, the arrow buttons are not displayed.  
- Go to My Site > Comments > comment details.
  - Verify only the Edit button is displayed.

| Arrows | No Arrows |
|--------|-------|
| arrows_shown | arrows_not_shown |

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature disabled.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift [hunk 1]
```diff
@@ -311,10 +344,30 @@ private extension CommentDetailViewController {
...
+        }
+
+        if comment.allowsModeration() {
+            barButtonItems.append(editBarButtonItem)
+        }
+
+        navigationItem.setRightBarButtonItems(barButtonItems, animated: false)
+    }
+
+    func navigationButtons() -> UIBarButtonItem {
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift]: Nitpick, but I'd either rename this to `makeNavigationButtons` or move it to a lazy var named something like `navigationButtonsStackView`.

**Review by frosty — COMMENTED** [review]
This works for me, however I'm seeing a weird animation in the buttons when the view appears:

https://user-images.githubusercontent.com/4780/152777985-556b5d9f-f1d9-4963-8c84-8e2475ab0513.mov

I'm also not sure if the button names and labels should be reversed. The up arrow will presumably show the notification above in the list, which would be the _next_ notification. The down arrow would show the notification below, which chronologically would be the previous notification.

**Comment by ScoutHarris** [comment]
> The up arrow will presumably show the notification above in the list, which would be the next notification...

I was thinking up arrow is previous in the list (hence `previousButton`), but I can see it also means next notification. I suppose it depends on how it's interpreted. Anyway, it doesn't matter to me, I've changed it.

> a weird animation in the buttons when the view appears

Eagle eyes. :smile: I removed a redundant call to `configureNavigationBar` which seems to have resolve that.

Ready for another go please!

**Comment by jkmassel** [comment]
You can test the <strong>Jetpack</strong> changes on this Pull Request by downloading it from AppCenter <a href='https://install.appcenter.ms/orgs/automattic/apps/jetpack-installable-builds/'>here</a> with build number: <code>pr17897-86be2e5</code>. IPA is available <a href='https://appcenter-filemanagement-distrib4ede6f06e.azureedge.net/2a5f9ae2-f6f6-4162-8f3f-d590aa70f711/Jetpack%20Alpha.ipa?sv=2019-02-02&sr=c&sig=6LlZo%2FLUu5jS1l5oxFE5jizN39IeXceMEANLD9rCW58%3D&se=2022-02-08T22%3A20%3A27Z&sp=r'>here</a>. If you need access to this, you can ask a maintainer to add you.

**Comment by jkmassel** [comment]
You can test the <strong>WordPress</strong> changes on this Pull Request by downloading it from AppCenter <a href='https://install.appcenter.ms/orgs/automattic/apps/WPiOS-One-Offs/'>here</a> with build number: <code>pr17897-86be2e5</code>. IPA is available <a href='https://appcenter-filemanagement-distrib4ede6f06e.azureedge.net/656f95b4-13bf-43f0-8b58-3822d5417bdc/WordPress%20Alpha.ipa?sv=2019-02-02&sr=c&sig=tBchGsBznlRcz3UWJoIjLNMEqmjosFE3iurfHD%2FeYB4%3D&se=2022-02-08T22%3A21%3A57Z&sp=r'>here</a>. If you need access to this, you can ask a maintainer to add you.

**Review by frosty — APPROVED** [review]
:shipit:

**Comment by ScoutHarris** [comment]
Thanks @frosty!

**Pull request merged**